packages/screens/AdministrationDashboard/component/GuardiansBox.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState } from "react";
+import {
+  FlatList,
+  Image,
+  ImageBackground,
+  TouchableOpacity,
+  View,
+  useWindowDimensions,
+  ViewStyle,
+} from "react-native";
+
+import avaPNG from "../../../../assets/default-images/ava.png";
+import guardianPng from "../../../../assets/default-images/guardian_1.png";
+import addCircleSVG from "../../../../assets/icons/add-circle.svg";
+import checkBadgeSVG from "../../../../assets/icons/check-badge.svg";
+import dotSVG from "../../../../assets/icons/dot-more.svg";
+import downSVG from "../../../../assets/icons/down.svg";
+import trashSVG from "../../../../assets/icons/trash-white.svg";
+import upSVG from "../../../../assets/icons/up.svg";
+import { BrandText } from "../../../components/BrandText";
+import { SVG } from "../../../components/SVG";
+import { Box, BoxStyle } from "../../../components/boxes/Box";
+import { SearchInput } from "../../../components/sorts/SearchInput";
+import {
+  neutral33,
+  neutral00,
+  gradientColorLightBlue,
+} from "../../../utils/style/colors";
+import { fontSemibold14, fontSemibold16 } from "../../../utils/style/fonts";
+
+const MD_BREAKPOINT = 820;
+
+export const GuardiansBox = () => {
+  const { width } = useWindowDimensions();
+
+  const [guardiansHandler, setGuardiansHandler] = useState<number[]>([1, 2]);
+  const [isListOpen, setIsListOpen] = useState<number>();
```

Comment:
normally we prefix `is` for bool value, here I can see that we can have number value so maybe we should change isListOpen => openedList to avoid confusion